app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,4 @@
+class PagesController < ApplicationController
```

Comment:
Is there a reason we need a new controller here, rather than putting the contact action in the home controller?